mkdocs.yml

```diff
@@ -14,6 +14,8 @@ theme:
     scheme: "slate"
     primary: "teal"
     accent: "teal"
+  font:
+    text: Inter
```

Comment:
It's a subtle change (I had to check the new and the old version side by side), but it does look better! :star_struck: